packages/experimental/Avatar/src/NativeAvatar.tsx

```diff
@@ -0,0 +1,134 @@
+/** @jsx withSlots */
+import { compose, UseSlots, buildProps, mergeProps, withSlots } from '@fluentui-react-native/framework';
+import { ImageURISource, NativeModules, ViewProps, ColorValue } from 'react-native';
+import { ensureNativeComponent } from '@fluentui-react-native/component-cache';
+
+const avatarName = 'Avatar';
```

Comment:
Maybe change this to NativeAvatar as well? I think this affects what shows up in the React Tree when you inspect it with a react Inspector, and maybe some more

Reply:
Also affects Theme's component section and how it finds the components to affect.

Reply:
Fixed. Thanks for catching that